lib/irc/ConnectionInstance.js

```diff
@@ -18,6 +18,10 @@ const PING_TIMEOUT_MS = 1000 * 60 * 10;
 // due to throttling.
 const THROTTLE_WAIT_MS = 20 * 1000;
 
+// The minimum time to wait between connection attempts if the kernel has given us
+// a net_error.
+const NET_ERROR_WAIT_MS = 5 * 1000;
```

Comment:
This suggests we are likely to have success if we retry sooner than if we are throttled. It should be explained in the comments why we retry on net error, and why it is done sooner than if we are throttled (for example).